hack/build-go.sh

```
@@ -12,7 +12,8 @@ GOOS=${GOOS:-${GOHOSTOS}}
 GOARCH=${GOARCH:-${GOHOSTARCH}}
 
 # Go to the root of the repo
-cd "$(git rev-parse --show-cdup)"
+rel="$(git rev-parse --show-cdup)"
+[ $rel ] && cd $rel
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)